_posts/events/2018-08-28-propublica-parking-tickets.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: event
+categories:
+  - events
+links:
+title: "Parking Tickets and Bankruptcy"
+description: "If you're a Chicagoan, Jason Kunesh is your design director. What does that even mean? He's working to introduce product & service design disciplines and techniques to make sure the city is accessible, inclusive, and strategic when it launches tech products."
+speakers: Melissa Sanchez, ProPublica Illinois reporter, and David Eads, ProPublica Illinois news apps developer
+image: /images/events/315-propublica-parking-tickets.jpg
+image_credit:
+date: 2018-08-28T18:00:00-05:00
+event_id: 315
+youtube_id: U6GVI9LAOTg
+agenda: https://docs.google.com/document/d/1fywroAg8p4R6tzMAQ0R9niJRe3MZ2NVnfkfFR1zO9xU/edit?usp=sharing
+sponsor: Google
+asl_provided: true
+tags:
+ - propublica
```

Comment:
our tag taxonomy is a bit wild still, but i would code this as `journalism` instead of the name of the org